src/test/sql/rls_manual_integration_test.sql

```diff
@@ -30,53 +32,62 @@ INSERT INTO row_level_security_test_schema.rls_sales VALUES
+GRANT ALL PRIVILEGES TO RLS_USR_1;
 
+DROP USER IF EXISTS RLS_USR_2 CASCADE;
+CREATE USER RLS_USR_2 IDENTIFIED BY "RLS_USR_2";
+GRANT ALL PRIVILEGES TO RLS_USR_2;
 
---Create users and grant privileges
-CREATE USER rls_usr_1 IDENTIFIED BY "rls_usr_1";
-GRANT CREATE SESSION TO rls_usr_1;
-GRANT CREATE VIEW TO rls_usr_1;
-GRANT SELECT ON row_level_security_test_schema TO rls_usr_1;
-GRANT IMPERSONATE ANY USER TO rls_usr_1;
+DROP USER IF EXISTS RLS_USR_3 CASCADE;
+CREATE USER RLS_USR_3 IDENTIFIED BY "RLS_USR_3";
+GRANT ALL PRIVILEGES TO RLS_USR_3;
 
-CREATE USER rls_usr_2 IDENTIFIED BY "rls_usr_2";
-GRANT CREATE SESSION TO rls_usr_2;
-GRANT CREATE VIEW TO rls_usr_2;
-GRANT SELECT ON row_level_security_test_schema TO rls_usr_2;
-GRANT IMPERSONATE ANY USER TO rls_usr_2;
+DROP USER IF EXISTS RLS_USR_4 CASCADE;
+CREATE USER RLS_USR_4 IDENTIFIED BY "RLS_USR_4";
+GRANT ALL PRIVILEGES TO RLS_USR_4;
 
-CREATE USER rls_usr_3 IDENTIFIED BY "rls_usr_3";
-GRANT CREATE SESSION TO rls_usr_3;
-GRANT CREATE VIEW TO rls_usr_3;
-GRANT SELECT ON row_level_security_test_schema TO rls_usr_3;
-GRANT IMPERSONATE ANY USER TO rls_usr_3;
-
-CREATE USER rls_usr_4 IDENTIFIED BY "rls_usr_4";
-GRANT CREATE SESSION TO rls_usr_4;
-GRANT CREATE VIEW TO rls_usr_4;
-GRANT SELECT ON row_level_security_test_schema TO rls_usr_4;
-GRANT IMPERSONATE ANY USER TO rls_usr_4;
-
-
---Create rls users configuration table
-CREATE OR REPLACE TABLE row_level_security_test_schema.rls_users
+--Create rls users configure table
+CREATE OR REPLACE TABLE row_level_security_test_schema.exa_rls_users
     (
     exa_user_name VARCHAR(200),
     exa_role_mask DECIMAL(20,0)
     );
 
-INSERT INTO row_level_security_test_schema.rls_users VALUES ('rls_usr_1', NULL);
-INSERT INTO row_level_security_test_schema.rls_users VALUES ('rls_usr_2', 1);
-INSERT INTO row_level_security_test_schema.rls_users VALUES ('rls_usr_3', 3);
-INSERT INTO row_level_security_test_schema.rls_users VALUES ('rls_usr_4', 15);
+INSERT INTO row_level_security_test_schema.exa_rls_users VALUES ('RLS_USR_1', NULL);
+INSERT INTO row_level_security_test_schema.exa_rls_users VALUES ('RLS_USR_2', 1);
+INSERT INTO row_level_security_test_schema.exa_rls_users VALUES ('RLS_USR_3', 3);
+INSERT INTO row_level_security_test_schema.exa_rls_users VALUES ('RLS_USR_4', 15);
+INSERT INTO row_level_security_test_schema.exa_rls_users VALUES ('SYS', 15);
+
+--Create Virtual Schema
+CREATE CONNECTION jdbc_exasol_connection
+TO 'jdbc:exa:host:port'
+USER 'sys'
+IDENTIFIED BY 'password';
+
+--/
+CREATE OR REPLACE JAVA ADAPTER SCRIPT row_level_security_test_schema.adapter_script_exasol_rls AS
+    %scriptclass com.exasol.adapter.RequestDispatcher;
+    %jar /buckets/bfsdefault/rls/row-level-security-0.1.0.jar;
+    %jar /buckets/bfsdefault/rls/exajdbc.jar;
+/
+;
 
-SELECT * FROM row_level_security_test_schema.rls_users;
+CREATE VIRTUAL SCHEMA virtual_schema_rls USING row_level_security_test_schema.adapter_script_exasol_rls WITH
+  SQL_DIALECT     = 'RLS'
```

Comment:
Rename to EXASOLRLS

Reply:
I renamed it to EXASOL_RLS already. Can I use an underscore or should I remove it?

Reply:
Underscore is fine.